app/components/message_attachments/action_button/action_button.js

```diff
@@ -34,7 +34,19 @@ export default class ActionButton extends PureComponent {
                 containerStyle={style.button}
                 onPress={this.handleActionPress}
             >
-                <Text style={style.text}>{name}</Text>
+                <Markdown
+                    isEdited={false}
```

Comment:
Not related to this PR, but we really need to rethink how the `Markdown` component is structured since this is way more complicated than it should be.